apps/src/applab/setPropertyDropdown.js

```diff
@@ -53,10 +53,49 @@ var PROP_INFO = {
   step: { friendlyName: 'step', internalName: 'step', type: 'number' }
 };
 
+let DEFAULT_PROP_VALUES = {
+  width: '100',
+  height: '100',
+  x: '100',
+  y: '100',
+  'text-color': '"red"',
+  'background-color': '"red"',
+  'font-size': '100',
+  'text-align': '"left"',
+  hidden: 'true',
+  text: '"text"',
+  placeholder: '"text"',
+  image: '"https://code.org/images/logo.png"',
+  'icon-color': '"red"',
+  'group-id': '"text"',
+  checked: 'true',
+  readonly: 'true',
+  options: '["option1", "etc"]',
+  value: '100',
+  min: '100',
+  max: '100',
+  step: '100',
+};
+
 // When we don't know the element type, we display all possible friendly names
-var fullDropdownOptions = _.uniq(Object.keys(PROP_INFO).map(function (key) {
-  return '"' + PROP_INFO[key].friendlyName + '"';
-}));
+var fullDropdownOptions = _.uniqBy(Object.keys(PROP_INFO).map(function (key) {
+  if (PROP_INFO[key].alias) {
+    return;
+  }
+  let friendlyName = PROP_INFO[key].friendlyName;
+  let setValueParam = DEFAULT_PROP_VALUES[friendlyName];
+  if (setValueParam) {
+    return {
+      text: '"' + friendlyName + '"',
+      display: '"' + friendlyName + '"',
+      setValueParam: setValueParam,
+    };
+  } else {
+    return '"' + PROP_INFO[key].friendlyName + '"';
+  }
+}).filter(object => object), object => {
+  return object.text;
+});
```

Comment:
The iteratee you give to [`_.uniqBy`](https://lodash.com/docs/4.17.4#uniqBy) here looks like it only works for the new case where a default prop value exists for the key.  If you hit the `else` case on line 94 I think you'll be calling `.text` on a string, which will give `undefined` - probably not what you intended.

If the coverage information I'm seeing is correct, we also don't hit this case (line 94 specifically) in unit tests.  It'd be worth making there's a test for this case.